tests/test_constants.py

```diff
@@ -7,6 +8,21 @@
 
 
 class TestConstants(unittest.TestCase):
+    @staticmethod
+    def _exec(command):
+        process = os.popen(command)
+        output = process.read().strip()
+        process.close()
+        return output
+
+    @staticmethod
+    def _extract_current_app(dumpsys_output):
+        return TestConstants._exec('CURRENT_APP="' + dumpsys_output + '" && ' + constants.CMD_PARSE_CURRENT_APP + ' && echo $CURRENT_APP')
```

Comment:
```suggestion
        idx = constants.CMD_CURRENT_APP.find(constants. CMD_PARSE_CURRENT_APP)
        return TestConstants._exec('CURRENT_APP="' + dumpsys_output + '" && ' + constants.CMD_CURRENT_APP[idx:])
```

Reply:
This approach might work, but will not ensure the integrity of the tests in the future, also I prefer existing solution as it is more readable compared to trimming a command based on index.

Reply:
I'll leave this as is. If a future pull request modifies any of the constants used for obtaining the current app, I'll consider modifying this or adding additional tests at that time.